app/src/androidTest/java/com/github/sdp_begreen/begreen/activities/MainActivityTest.kt

```diff
@@ -121,6 +125,7 @@ class MainActivityTest {
                 `when`(db.getAllUsers()).thenReturn(listOf(user1))
                 `when`(db.getAllBins()).thenReturn(bins)
                 `when`(eventService.getAllEvents(RootPath.MEETINGS, Meeting::class.java)).thenReturn(flowOf())
+                whenever(eventService.getAllEvents(RootPath.CONTESTS, Contest::class.java)).thenReturn(flowOf())
```

Comment:
Shouldn't we use the same to keep coherence ? I did a small research and it seems that `whenever` is preferred to `when`, but I think that we should use one only one of them. What do you think about it ?

Reply:
Actually as I understood it, `whenever` is just a Kotlin wrapper around `when`, as `when` is also a keyword in kotlin, and it is a bit annoying to always have to write back quote around it. I didn't know about this method until recently and that's why there is a lot of `when` everywhere.

And I let the `when` for know as I didn't want to go through every test files to change it. But you are right, I will do it for `MainActivityTest`, and we will do it as we go for other tests files when we modify them.

Reply:
Great !